Xiaomi/Mi8/res/values/config.xml

```diff
@@ -103,13 +103,13 @@
     <fraction name="config_autoBrightnessAdjustmentMaxGamma">200.0%</fraction>
     <fraction name="config_maximumScreenDimRatio">29.999996%</fraction>
     <integer name="config_autoBrightnessBrighteningLightDebounce">2000</integer>
-    <integer name="config_autoBrightnessDarkeningLightDebounce">6000</integer>
+    <integer name="config_autoBrightnessDarkeningLightDebounce">2000</integer>
     <integer name="config_screenBrightnessDark">1</integer>
-    <integer name="config_screenBrightnessDim">1</integer>
+    <integer name="config_screenBrightnessDim">4</integer>
```

Comment:
Dim is no higher than config_screenBrightnessSettingMinimum
That sounds wrong.
Is this stock ROM value?

Reply:
Both of them are come from stock ROM but different Android version.
`1` froms Oreo based ROM, and `4` froms Pie based ROM
But I think I should changing it to `2`